contrib/depends/packages/unwind.mk

```diff
@@ -3,6 +3,11 @@ $(package)_version=1.2
 $(package)_download_path=http://download.savannah.nongnu.org/releases/libunwind
 $(package)_file_name=lib$(package)-$($(package)_version).tar.gz
 $(package)_sha256_hash=1de38ffbdc88bd694d10081865871cd2bfbb02ad8ef9e1606aee18d65532b992
+$(package)_patches=fix_obj_order.patch
+
+define $(package)_preprocess_cmds
+  patch -p0 < $($(package)_patch_dir)/fix_obj_order.patch
+endef
```

Comment:
Please add /monero#6118 to the end of the commit message.

Reply:
Resolved.